cmd/provider/main.go

```diff
@@ -91,6 +97,13 @@ func main() {
 		Features:                &feature.Flags{},
 	}
 
+	if *enableManagementPolicies {
+		o.Features.Enable(feature.EnableBetaManagementPolicies)
+		log.Info("Beta feature enabled", "flag", feature.EnableBetaManagementPolicies)
+	}
+
+	kingpin.FatalIfError(ctrl.NewWebhookManagedBy(mgr).For(&v1alpha1.Object{}).Complete(), "Cannot create Object webhook")
```

Comment:
> ...For(&v1alpha1.Object{})...

I assume it doesn't make any difference if we use `v1beta1.Object{}` or we have to start a webhook for it as well. Could you double check and may be leave a code comment accordingly?

Reply:
I can see no difference, checked briefly the code, and did tests with both. Left a comment.